pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -751,14 +757,19 @@ public static Map<String, List<KitRequestShipping>> getAllKitRequestsByRealm(@No
     }
 
     private static PreparedStatement getPreparedStatement(@NonNull Connection conn, String target, @NonNull String realm, String type,
-                                                          boolean getAll) throws SQLException {
+                                                          Integer kitTypeId, boolean getAll) throws SQLException {
         PreparedStatement stmt = null;
         if (getAll) {
             String query = SQL_SELECT_KIT_REQUEST.concat(QueryExtension.BY_REALM);
             stmt = conn.prepareStatement(query);
             stmt.setString(1, realm);
         } else {
-            if (StringUtils.isNotBlank(realm) && StringUtils.isNotBlank(type)) {
+            if (kitTypeId != null && kitTypeId != null) {
```

Comment:
Is this the same criteria in both clauses?

Reply:
😂 I don't know what happened there, thanks for catching it